### PR TITLE
updated style of job-list

### PR DESCRIPTION
**Pull request by ThePeach**

### CODE REVIEW

**Comment by danbull** [comment]
I really think the CTA is being overused here; it's losing it's role as a CTA and is just becoming the default style for a link on this page.  The office location could be a link maybe? Can discuss!

Also maybe instead of repeating "full time" this could be assumed unless it says otherwise? e.g. (Contract)

File: src/ui-lib/sass/05-components/03-organisms/_job-list.scss [hunk 1]
```diff
@@ -1,12 +1,29 @@
...
+  }
+
+  > header > p {
     @include typi(small-text);
-    margin-top: $grav-sp-m;
+    margin-top: 0;
   }
 
   > ul {
     padding-left: $grav-sp-m;
```
Comment (danbull) [inline src/ui-lib/sass/05-components/03-organisms/_job-list.scss]: I think this should be `0` on mobile too?

Comment (ThePeach) [reply src/ui-lib/sass/05-components/03-organisms/_job-list.scss]: correct!!!!

**Comment by ThePeach** [comment]
very good points, and to reply to that:

1) we don't really have anything to link to that, I'm going to remove that, and I do agree with you, it should be a normal link, not a CTA.
2) the full-time etc, maybe that's something we can check when doing the user test, as I think we would have the same problem if all the positions were contracts.

**Pull request merged**